fapi-client/src/main/scala/com/gu/facia/api/utils/CardStyle.scala

```diff
@@ -0,0 +1,101 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.Content
+import com.gu.facia.api.utils.ContentApiUtils._
+import com.gu.facia.client.models.MetaDataCommonFields
+
+object CardStyle {
+
+  def isExternalLink(content: Content, metaDataFields: MetaDataCommonFields): Boolean = (for {
```

Comment:
This function is only used in one place and has identical arguments to the function that calls it. It's not tested or anything so does it need to be here separately?